resources/api/reservation.py

```diff
@@ -541,6 +543,13 @@ def _can_reserve_resource_with_current_login(self, request, resource):
         if resource_authentication in ('none', ''):
             return True
 
+        if (
+            resource.is_admin(request.user) or
+            resource.is_manager(request.user) or
+            resource._has_perm(request.user, 'can_make_reservations')
```

Comment:
Maybe add a comment here why this only concerns officials/managers/admin and not regular users (you'd think regular users also have the "can make reservations" permission). At least it wasn't that clear to me.